### DISCUSSION TITLE
Apktool not work in termux

### DISCUSSION THREAD

**Discussion by Lohita9** [termux/termux-packages#15045] [General]
![Screenshot_2023-02-09-07-12-21-341_com termux](https://user-images.githubusercontent.com/123667347/217695040-7b335402-740f-4469-9c64-45ada5b9b8cb.jpg)

**Comment by sylirre** [comment]
`apktool` package has been disabled due to runtime issues and Termux no longer provides it for installation.

***

Our current build scripts are there: https://github.com/termux/termux-packages/tree/0c6662897f0e4dcbe2ae3fff0115c30a99c0fbd9/disabled-packages/apktool. If you can fix the package so it will work in Termux, please submit necessary changes and we will make `apktool` available again.

**Comment by Lohita9** [comment]
@sylirre Please available apktool for 1 hour.

🙏🙏🙏
